clients/python/requirements_dev.txt

```diff
@@ -15,4 +15,5 @@ oauthlib==3.1.0
 requests_oauthlib==1.2.0
 requests==2.22.0
 thrift_connector==0.24
-thrift==0.11.0
\ No newline at end of file
+thrift==0.11.0
+configparser==4.0.2
```

Comment:
Is this needed? `configparser` is builtin. Is this for Python 2 clients?

Reply:
Yes, configparser is in builtin in python. It was renamed to configparser in python 3 from Configparser in python2. Since we are only supporting python3. This is unnecessary, removing it from requirements.